---
title: December 2022 Newsletter
publishDate: "2022-12-22"
---

## Welcome word from the co-administrators

Dear member of the Metaproteomics Initiative,

After a bit more than a year with an official executive board and a long process to get things up and running, it was about time we came back to you, with interesting and hopefully relevant news for everybody.

We have set up this newsletter to improve information transfer between the executive board and the community, but also to stimulate interaction among the community members. The newsletter will include digests from our monthly meetings, summaries of metaproteomics sessions at (international) conferences, upcoming events such as symposia and workshops, job opportunities, and members' research blog posts. Therefore, we would love to help you to share your information too! So please, send us the content you'd like to see in the next newsletter at info@metaproteomics.com

We look forward to reading more from you!

Benoit Kunath and Tim Van Den Bossche

---

## Executive Board Digest

Among the different tasks the board currently undertakes, the main topics are the organization of the next International Metaproteomics Symposium in April 2023 in Avignon, France, and the organization of follow-up multi-laboratory projects.

A few years ago, the first Critical Assessment of Metaproteome Investigation (CAMPI) was initiated at the 3rd symposium in Leipzig and led to the first community-driven, multi-laboratory comparison in metaproteomics. However, the scope was very broad, and many questions remained unanswered. Thus, the Metaproteomics Initiative and its scientific committee has been brewing several other, more specific inter-laboratory projects to tackle those different questions. We hope that many of you will be able to participate in those projects.

---

## CAMPI-2 Open call

Sample handling is one of the major challenges in metaproteomics research. Directly after sample collection, the taxonomic composition and the functioning of the ecosystem might be influenced by sample storage and transportation (including exposure to temperature, humidity, pH, O2, etc).

Here, we launch CAMPI-2, focusing on methods for preserving metaproteome profile of a given sample before microbiota analyses.

We are looking for volunteering labs who will apply their methods to stabilize a fecal and/or soil sample. The idea is to assess the effect of “storage” and “transportation” on the microbiome sample. The expected CAMPI-2 result is to provide a standard procedure for sample handling and transportation. [Find out more](https://metaproteomics.org/campi/campi2/)

---

## CAMPI-3 Taxonomic and Functional Annotation of Metaproteomics Data

Metaproteomics plays a crucial role in characterizing microbiome functions. The acquired MS/MS data is searched against a protein sequence database to identify peptides, which are then used to infer a list of proteins in a metaproteome sample. Protein inference remains a major challenge for metaproteomics because of the many degenerate peptides shared among homologous proteins in different organisms.

The CAMPI-3 project will reach out to metaproteomics researchers with microbial datasets of low and high complexity and seek answers regarding the annotation of taxonomy, protein inference, and functional annotation.

CAMPI-3 will be carried out in four phases: A) Sample Preparation, B) Pre-Analysis, C) Promotions, and D) Analysis phase. Following their preparation, a microbial dataset from a synthetic mixture (low-complexity sample) and a high-complexity gut microbiome/wastewater dataset will be pre-analyzed to assess the data quality and define the metrics to report taxonomic and functional annotation results. Then the study will be promoted and both datasets provided, along with guidelines and points to be addressed, to the participating labs. In the last stage, the CAMPI-3 team will analyze the study participants' submissions and compare with the ground truth data (for the low-complexity data) and methods and results for the high-complexity dataset. Currently, CAMPI-3 is in the sample preparation phase. We anticipate it to be completed by the end of 2023, with the outcome of the study made available to the microbiome community.

---

## Conference Digest

### One Small Step for Man, One Giant Leap for Metaproteomics. Metaproteomics Matters at HUPO 2022

Metaproteomics researchers from all over the world reached the shores of Mexico to attend and present their work at the HUPO 2022 in Cancún (Mexico).

The HUPO 2022 Congress had a session on the importance of metaproteomics in human biology. Jean Armengaud from CEA (France) presented the keynote talk which underlined the importance of metaproteomics, a powerful methodology that gives fine-grain taxonomic description of the biological sample and, more importantly, a true functional characterization of the identified taxa and the host. He also outlined the possible application of the methodology for clinical diagnostics and introduced the numerous HUPO attendees to the Metaproteomics Initiative, a remarkable networking tool. The keynote talk was followed by a talk by Zhibin Ning (University of Ottawa) who presented on the iMetaLab platform. The iMetaLab suite is a actively developed toolset aiming to provide a one-top metaproteomics workflow, with quantification fully supported to aid comprehensive taxon and function analysis. Yannek Nowatzky from Bundesanstalt für Materialforschung und -prüfung (BAM) presented his work on Mistle which is the first spectral library search program developed for metaproteomics. The presentation demonstrated that predicted spectral libraries can enhance peptide identification for metaproteomics and Mistle provides the means to efficiently search large-scale spectral libraries. Lastly, Pratik Jagtap from University of Minnesota (Galaxy-P Team) presented the collaborative work on clinical metaproteomics to detect microbial peptides from clinical samples. He described a bioinformatics workflow for processing of microbial and host proteins, generating verified microbial peptide candidates suitable for targeted analysis within individual patient samples. The talks at the session chaired by José Ángel Huerta Ocampo and Pratik Jagtap, were followed by a lively questions and answers session and follow up discussion. Apart from this there were multiple posters on metaproteomics highlighting the importance of data acquisition and bioinformatics to measure the microbiome dynamics in the context of human biology.

Jean Armengaud was also invited to present the current status of human metaproteomics during the HPP Day, held the day after HUPO 2022. This specific symposium was an important step for the ongoing reflection on the future project “A function for each protein”. To add the cherry onto the cake, Tim Van Den Bossche was chosen onto the HUPO Council as a member. This will greatly enhance the partnership between the Metaproteomics Initiative and the Human Proteome Organization.

In general, this was a successful first step for the metaproteomics community and will pave the way for more human-centric microbiome research using mass spectrometry methods.

---

## Save the Date

### The 5th International Metaproteomics Symposium

We are delighted to announce that [the 5th Metaproteomics Symposium](http://ims23.com/) will be taking place on 25–27 April 2023 in Avignon, France. The symposium welcomes international scientists from every career stage and offers plenty of opportunities for interaction on the most important topics related to metaproteomics and microbiome studies.

Check out our exciting [program](https://metaproteomics.org/IMS5.pdf) and stay tuned for more details!

---

## Metaproteomics articles you don't want to miss

### The invisible Part in Metaproteomics

The human gut microbiota has been extensively studied using multiple omics technologies. Although metagenomics reveals that hundreds of bacteria reside in the human gut, the dynamic functions of microbiota are primarily driven through proteins. Mass spectrometer-based proteomics, which directly measures proteins, can assess functional alterations in the microbiota.

Data-dependent acquisition (DDA) is extensively used in metaproteomics. However, in DDA the top N most abundant precursor ions are selected for fragmentation, and the other precursors, which are also often peptide candidates, are neglected. Unfortunately, we do not even know how many peptides we are missing.

Haonan Duan (Ph.D. student) et al., from the NorthOmics group led by Daniel Figeys, have assessed the detection limit of the metaproteomics in DDA mode. They spiked 15N-labeled Escherichia coli peptides at different percentages into peptides derived from the human gut microbiota. As they reported in Analytical Chemistry, bacteria whose biomass in the microbiota is below 0.5% can not be identified in their workflow.

The isotopically labeled peptides are distinguished from the regular peptides in mass spectra. This enabled them to only detect the E. coli peptides, the number of which can be correlated with the spiked-in levels. They found that the number of the identified peptide decreased linearly with the spiked-in level. Over 20,000 peptides can be identified in pure E. coli samples, while when the biomass percent is down to 0.5%, no more than two hundred peptides can be identified. They further used the match-between-run strategy to extract peptide candidates from MS1 spectra. Over 1500 peptides with high reliability can be retrieved even at 0.5% level.

The NorthOmics group clarified the detecting limit of the widely used DDA-metaproteomics. They are further working on increasing the depth of metaproteomics using methods such as data-independent acquisition (DIA) and other methods.

### Adaptation of a microbial community to demand-oriented biological methanation

Biological conversion of the surplus of renewable electricity and carbon dioxide (CO2) fom biogas plants to biomethane (CH4). To investigate the robustness of the biomethanation process regarding power oscillations, a biomethanation reactor was exposed to five hydrogen (H2)-feeding regimes lasting several days. Analyses of the taxonomic composition of the microbiome revealed a high abundance of the bacterial phyla Firmicutes, Bacteroidota and Thermotogota followed by hydrogenotrophic Archaea of the phylum Methanobacteriota. Homo-acetogenic and heterotrophic fermenting Bacteria formed a complex food web with methanogens. The abundance of the methanogenic Archaea roughly doubled during discontinuous H2-feeding, which was related mainly to an increase in acetoclastic Methanothrix species. Results also suggested that Bacteria feeding on methanogens could reduce overall CH4 production. On the other hand, using inactive biomass as a substrate could support the growth of methanogenic Archaea. During the BM process, the additional production of H2 by fermenting Bacteria seemed to support the maintenance of hydrogenotrophic methanogens at non-H2-feeding phases. Besides the elusive role of Methanothrix during the H2-feeding phases, acetate consumption and pH maintenance at the non-feeding phase can be assigned to this species. [Find out more](https://biotechnologyforbiofuels.biomedcentral.com/articles/10.1186/s13068-022-02207-w)
