---
title: October 2023 Newsletter
publishDate: "2023-10-27"
---

## From the co-administrators

Dear members of the Metaproteomics Initiative,

We are delighted to bring you the fourth edition of our newsletter!
In this newsletter, we provide you with CAMPI status updates and research highlights contributed by members like you! Importantly, we’re also switching from communication platforms - more specifically, we are moving away from Basecamp to Slack. We hope this platform will be more suited to your needs to collaborate and communicate with the other members of our Initiative.

We continue to be grateful for the content you share, and we encourage you to reach out to us if you would like your research to be featured in future newsletters. Your continued support and active participation in the Metaproteomics Initiative are greatly appreciated.

We hope that many of you can join us and engage in discussions regarding the Initiative and its projects.

Kind regards,
Tim Van Den Bossche and Benoit Kunath

---

## Executive Board Digest

### Introducing the Metaproteomics Initiative Slack Workspace

It's our pleasure to announce our new Slack Workspace for the Metaproteomics Initiative! We're setting up this workspace to bring everyone a more dynamic and interactive platform for discussions and collaborations. Join us by clicking [here](https://metaproteomics.us21.list-manage.com/track/click?u=b22627513ce4d06ccc1c39cb8&id=e059ff0920&e=028934b95c) to engage with the community, share your research, and stay updated on upcoming events and announcements all in one place. Later this month we'll send a direct invite as well!


We're excited to see you on Slack!

### New membership form for labs

Since this month, we have made some significant updates to our membership form after receiving feedback from several members. In this revamped form, we now made a clear distinction between two membership categories: individual membership, and lab membership. As a recipient of this newsletter, you are already registered as an individual member - which is great! If your lab is (partly) involved with metaproteomics, you can also freely list your research group on our website. To get your lab listed (if it's not there already), all you need to do is fill in our new lab membership form.

### CAMPI-2: Sample handling

[CAMPI-2](https://metaproteomics.org/campi/campi2/) has led to a collaboration between 10 laboratories in Europe and the US, where five protocols for metaproteome stabilization are being compared. These protocols have been designed for two different microbiota samples: human feces and soil. This will provide a standard procedure for sample handling, one of the main challenges in metaproteomics studies.

Here's what's been happening:
The CAMPI-2 participants have successfully stabilized the entire sample set, including soil and stool samples, using the different protocols.
We are now moving on to the next crucial step: sample processing, which involves protein extraction and digestion. We aim to complete this phase by the end of July.

### CAMPI-3: Taxonomic and Functional Annotation of Metaproteomics Data

[CAMPI-3](https://metaproteomics.org/campi/campi3/), the latest study initiated by the Metaproteomics Initiative, has been launched.

The CAMPI-3 study aims to:

- Identify strengths and limitations of the bioinformatic workflows for protein assignment and peptide spectral matching quality among users at varying levels of expertise and experience in the field of metaproteomics.

- Evaluate the impact of key decision points that are commonly encountered across bioinformatic methods on taxonomic and functional annotation.

To facilitate your participation, we have prepared two datasets: a ground-truth “synthetic community” dataset and a biological dataset. These MS datasets and associated protein databases will serve as the foundation for your analyses. To get involved, we encourage you to download the data from [https://metaproteomics.org/campi3/download](https://metaproteomics.org/campi3/download) and leverage the provided reporting templates and metrics to ensure standardized sharing of results.

We value your contributions and understand the importance of maintaining confidentiality. Rest assured that all submissions will be treated with the utmost confidentiality and kept anonymous throughout the study.

If you have any questions or require further information about CAMPI-3, please visit [z.umn.edu/campi3faqs](http://z.umn.edu/campi3faqs). Moreover, the project coordinators (pjagtap@umn.edu, peterssl@ornl.gov and benoit.kunath@uni.lu) are ready to assist you at every step of the process. Don't hesitate to reach out to them.

Ready to submit your results? Please contact Anonymizer at ningzhibin@gmail.com

---

## Recap of the 5th Metaproteomics Symposium: Unlocking Microbiome Functions

The historical Popes’ Palace in Avignon, France, set the stage for the highly anticipated [5th Metaproteomics Symposium](https://ims23.com/), held on April 25-27, 2023, under the auspices of EuPA. This remarkable event brought together 110 participants from around the world to share their ground-breaking discoveries and innovations in the field of metaproteomics. Under the theme "Unlocking microbiome functions", the symposium served as a dynamic platform for advancing our understanding of microbial communities and their intricate functioning.

The symposium commenced with an enlightening overview by Robert Hettich, who provided valuable perspectives on the future impact of metaproteomics in microbiome research. The captivating keynote presentations further underscored the vast potential of metaproteomics. Notably, Ian Lidbury delved into the exploration of major plant-microbe interactions occurring in soil, shedding light on their crucial significance. Additionally, Veronika Pettersen focused on host-microbiome interactions in early life, specifically highlighting the role of gut fungi. These insightful talks demonstrated the tremendous progress made in the field and showcased the immense potential of metaproteomics in addressing pressing scientific challenges. Daniel Figeys further broadened the horizons presenting the prospects of metaproteomics in clinical research and drug development.

The symposium featured five different sessions dedicated to unraveling the intricate dynamics of microbial communities in various fields, emphasizing the role of metaproteomics in deciphering complex community interactions. State-of-the-art technologies driving metaproteomics research forward were showcased, including the exploration of DIA methods and advanced bioinformatics tools specifically designed for metaproteomics analysis.

Beyond the scientific sessions, the symposium fostered a spirit of collaboration and partnership, made possible within the Metaproteomics Initiative framework. Attendees had the opportunity to engage in interactive sessions, present posters, and participate in networking events, facilitating vibrant discussions and knowledge sharing. Dedicated roundtable discussions offering a prospectus on a ‘Grand challenge’ and ‘Metadata standards and Repository’ provided a platform to highlight emerging opportunities for the metaproteomics community and address its needs. A session dedicated to the CAMPIs benchmarks allowed participants to gain deeper insights into these studies and their goals while providing input for future [CAMPI](https://metaproteomics.org/campi/) editions.

The 5th Metaproteomics Symposium concluded with an air of enthusiasm, leaving attendees with new insights, potential collaborations, and a renewed passion for metaproteomics research. We extend our heartfelt gratitude to the speakers, sponsors, and participants who contributed to the resounding success of the symposium.

---

## Houston, We Have a (Microbiome) Problem… (Notes from Metaproteomics@ASMS2023)

The [71st edition of the annual conference](https://www.asms.org/conferences/annual-conference) for the American Society for Mass Spectrometry (Houston, June 4-8, 2023) which was attended by more than 6000 researchers, also had poster presentations, talks, workshop and a bioinformatics hub session on metaproteomics.

Subina Mehta (University of Minnesota) presented talk on ovarian cancer metaproteomics and Pierre-Olivier Schmit (Bruker Daltonique) presented talk on and DIA-PASEF for metaproteomics. Researchers from Pacific Northwest National Laboratory (Mary Lipton, Paul Piehowski, Kristin Burnum-Johnson), Oakridge National Laboratory (Robert Hettich, Samantha Peters), Ghent University (Tim Van Den Bossche, Tanja Holstein) and University of Minnesota (Pratik Jagtap, Timothy Griffin, Andrew Rajczewski) presented posters and attended the workshop on “Houston, We Have a Microbiome Problem (...and how the Metaproteomics Initiative aims to solve it)” and Bioinformatics Hub on “Metaproteomics”.

The [Wednesday evening workshop](https://drive.google.com/file/d/1BOE61dgbx-U_Euw3XkeTKfR4-W8mL1bG/view) was attended by almost 50 attendees. The in-person attendees were welcomed by Timothy Griffin, highlighting the importance of mass spectrometry based metaproteomics research. This introduction was followed by Tim Van Den Bossche, who introduced the Metaproteomics Initiative and invited the attendees to join the various activities of the Initiative. Bob Hettich presented the updates from the International Metaproteomics Symposium and also the rationale behind the CAMPI Study.

These presentations were followed by a presentation by Pratik Jagtap announcing the [CAMPI3 study](https://metaproteomics.org/campi/campi3/). Samantha Peters took part in the discussion and answered questions about some of the details of the CAMPI-3 sample types. Attendees also asked questions regarding plans for the future of Metaproteomics Initiative. The presenters and audience had a lively discussion that covered multiple aspects of metaproteomics ranging from sample preparation to mass spectrometry acquisition to bioinformatics analysis and biological interpretation. Active recruitment identified about 10 attendees who expressed interest in participating in the CAMPI-3 bioinformatics campaign. At the conclusion of the workshop, about a dozen participants hung around for more detailed and personalized discussions.

The participants were invited to the Bioinformatics Hub on Metaproteomics at the ASMS on Thursday morning for more expanded discussions on these and related topics. At the Bioinformatics Hub, attendees expressed an interest in a “Beginners Guide to Metaproteomics” manuscript.

---

## Save the Date

### 6th International Metaproteomics Symposium

We're thrilled to announce that the next International Metaproteomics Symposium will be held in the stunning Norway. Save the date for this exciting event, scheduled for January 2025!

Stay tuned for further updates.

---

## Metaproteomics articles you don't want to miss

### Our Gut Bugs Function Redundantly

Diversity matters, so does redundancy. Functional redundancy (FR) - the difference between taxonomic and functional diversity, describes the extent to which taxonomically distinct organisms can contribute to ecosystem functioning in similar ways.

FR has been shown to be a common event in the human gut microbiome. However, most previous studies of FR in the human microbiome have been conceptual rather than quantitative. The recent advances in deep metaproteomic techniques allow us to construct a protein-content network of human gut microbiomes and quantify the FR on the proteome level (i.e. FRp).

What does the FRp metric imply? A decrease in FRp may be associated with impaired microbiome stability and resilience. We show that naturally occurring protein-content network of gut microbiomes can preserve high taxonomic richness and high FRp at the same time, while environmental factors such as specific xenobiotics (e.g. antibiotics) and inflammatory bowel disease can significantly decrease the FRp of an individual’s microbiome.

We still have not fully understood the phenomenon of high FRp in our gut microbiome. What does it look like in some other diseases? How does it relate to the resistance and resilience of our microbiota? Can we leverage FRp to combat gut microbiome dysbiosis and diseases? Metaproteomics will continue to facilitate these exciting future explorations, since it opens the door to deepening the understanding of our microbiome functional ecology by examining the proteins – the implementers of microbiome functions.

Read the paper: [https://www.nature.com/articles/s41467-023-39149-2](https://www.nature.com/articles/s41467-023-39149-2)

### DIA-PASEF opens new frontiers for metaproteomics

[David Gómez-Varela's group](https://pharmtox.univie.ac.at/people/senior-scientist/dr-gomez-varela/) at the University of Vienna presented the first report using DIA-PASEF in metaproteomics. The [data](https://www.biorxiv.org/content/10.1101/2023.03.16.532922v1) showed very significant improvements in protein and peptide quantification depth, sensitivity, and reproducibility in complex gut mouse samples. Application to a preclinical disease model of chronic pain revealed new host-microbiome interactions leading to potential clinical interventions.

Building on this first work, [new data](https://www.bruker.com/en/products-and-solutions/mass-spectrometry/timstof/timstof-ultra.html) from Gómez-Varela's group presented at ASMS 2023  showed the possibility to quantify >45,000 protein groups and hundreds of species and functional pathways from a single LC-MS experiment. They were able to do this using sub-nanomolar peptide amounts (proteins and species could be detected even with only 10 picograms!) and in a short 22-minute effective LC gradient.

The group is actively working on new methodological and bioinformatic developments to push the limits towards deep, reproducible, and ultra-fast profiling of hundreds of samples per day. These new levels of sensitivity could eventually enable single-cell-type analysis in metaproteomics.

### Mix24X, a lab-assembled reference

Our team has proposed [Mix24X](https://pubmed.ncbi.nlm.nih.gov/37239979/), a new reference dataset to evaluate interpretation procedures for tandem mass spectrometry proteotyping of complex samples, such as microbiomes. This reference is an artificial assembly of individual peptidomes from 24 bacterial reference species.

Mix24X encompasses a wide range of microbial diversity, covering 20 distinct genera and 5 bacterial phyla. Notably, it includes challenging cases such as Shigella flexneri and Escherichia coli, which exhibit significant phylogenetic similarity, as well as several highly sequenced clades and distantly related species. Although Mix24X does not fully replicate  the complexity of a true microbiome sample, which comprises numerous species and challenging extracellular matrices, it can help to understand how your data processing pipeline performs in terms of sensitivity and accuracy.

To facilitate ease of use, Mix24X is readily available for download from the PRIDE repository. This dataset can be used to improve bioinformatic tools dedicated to proteotyping microorganisms from complex samples, or to extracting taxonomic or functional information from metaproteomic experiments.
