---
title: October 2023 Newsletter
publishDate: "2023-10-23"
---

## From the co-administrators

Dear members of the Metaproteomics Initiative,

We are delighted to bring you the fourth edition of our newsletter!
In this newsletter, we provide you with CAMPI status updates and research highlights contributed by members like you! Importantly, we’re also switching from communication platforms - more specifically, we are moving away from Basecamp to Slack. We hope this platform will be more suited to your needs to collaborate and communicate with the other members of our Initiative.

We continue to be grateful for the content you share, and we encourage you to reach out to us if you would like your research to be featured in future newsletters. Your continued support and active participation in the Metaproteomics Initiative are greatly appreciated.

We hope that many of you can join us and engage in discussions regarding the Initiative and its projects.

Kind regards,
Tim Van Den Bossche and Benoit Kunath

---

## Executive Board Digest

### Introducing the Metaproteomics Initiative Slack Workspace

It's our pleasure to announce our new Slack Workspace for the Metaproteomics Initiative! We're setting up this workspace to bring everyone a more dynamic and interactive platform for discussions and collaborations. Join us by clicking [here](https://metaproteomic.slack.com/join/shared_invite/zt-229pjpdg4-CV2z9UB18d3sDz89cY4_ew#/shared-invite/email) to engage with the community, share your research, and stay updated on upcoming events and announcements all in one place. Later this month we'll send a direct invite as well!

We're excited to see you on Slack!

### New membership form for labs

Since this month, we have made some significant updates to our membership form after receiving feedback from several members. In this revamped form, we now made a clear distinction between two membership categories: individual membership, and lab membership. As a recipient of this newsletter, you are already registered as an individual member - which is great! If your lab is (partly) involved with metaproteomics, you can also freely list your research group on our [website](https://metaproteomics.org/groups/). To get your lab listed (if it's not there already), all you need to do is fill in our new lab [membership form](https://docs.google.com/forms/d/e/1FAIpQLSdZUBIRB1QzDkcyzA1H1PWCUmucne3JVWcBwPucNzUGDTDFXQ/viewform).

---

## CAMPI updates

Both Critical Assessment of MetaProteome Investigation (CAMPI) are moving ahead. Here's the latest update.

### CAMPI-2: Sample handling

[CAMPI-2](https://metaproteomics.org/campi/campi2/) has led to a collaboration between 10 laboratories in Europe and the US, where five protocols for metaproteome stabilization are being compared. These protocols have been designed for two different microbiota samples: human feces and soil. This will provide a standard procedure for sample handling, one of the main challenges in metaproteomics studies.

CAMPI-2 is close to the final experimental step: data analysis!
We aim to complete this phase by November 2023. Soon after, the conclusive data will be shared with the whole metaproteomics community via open access journals.


### Calling Out CAMPI-3 enthusiasts!

[CAMPI-3](https://metaproteomics.org/campi/campi3/) aims to:

- Identify strengths and limitations of the bioinformatic workflows for protein assignment and peptide spectral matching quality among users at varying levels of expertise and experience in the field of metaproteomics.

- Evaluate the impact of key decision points that are commonly encountered across bioinformatic methods on taxonomic and functional annotation.

The deadline for submissions is November 5th

- Complete submissions to the two datasets will result in authorship on the planned CAMPI3 manuscript.
- Interested in participating? Sign up using the following link: z.umn.edu/campi3ing
- FAQs: z.umn.edu/campi3faqs

---

## Save the Date

### The Microbiome Wizards and Muggles Come Together in the Spring of 2024!

The ‘Deciphering Microbiome Functions’ workshop will bring together the microbiome community for a week of sharing, learning, and collaboration. The workshop will have themes for each day starting with Introduction (Day One), Metagenomics & Metatranscriptomics (Day Two), Metaproteomics & Metabolomics (Day Three), Collaboration, and Future Directions (Day Four). Each workshop day will include an introduction to the field by thought leaders, tutorials by experts, and brainstorming sessions in the afternoon.

Check the program [here](https://www.lorentzcenter.nl/site/index.php?pntHandler=WorkshopTemplatePage&pntType=ConPagina&id=1929) and reach out to organizers if you would like to be in Leiden in March 2024!

---

## Metaproteomics articles you don't want to miss

### A Galaxy of Informatics Resources for MS-based Proteomics

Mass spectrometry (MS)-based methods in proteomic research generate complex data types that are computationally challenging for their processing, analysis, and interpretation. The Galaxy bioinformatics ecosystem provides a flexible and scalable resource for research scientists to access software analysis tools, process data, and visualize results. The provenance tracking in Galaxy allows users to save and share complete analysis histories and workflows in proteomics analysis.  Current Galaxy workflows have enabled research in various fields such as pandemic research, proteogenomics, metaproteomics, MS-imaging, and MS-based proteomic clinical studies.

The Galaxy platform has been a leading platform for metaproteomics such as cervical-vaginal microbiome analysis; the Broncho-alveolar lavage fluid metaproteome in Acute Respiratory Failure; metaproteomics analysis of COVID-19 patient samples for secondary infections; role of proteases in irritable bowel syndrome; and metaproteomics analysis of the oral microbiome in lung cancer. Galaxy-based software and workflows have also been used in the Metaproteomics Initiative’s Critical Assessment of MetaProteome Investigation (CAMPI) and an inter-laboratory comparison study as part of the ocean metaproteomics community initiative. The Arntzen lab has developed Galaxy-based metaomics workflows to integrate metagenomics, metatranscriptomics, and metaproteomics and visualize through a web application, ViMO.

Galaxy ecosystem also offers access to training resources via the Galaxy training network (GTN) which promotes awareness and empowers adoption of these tools by the research community. The GTN site, offers a listing of proteomics resources along with other applications. The GTN is also used for workshops and the Galaxy-P Team has presented twenty-two metaproteomics-related workshops over the last six years by leveraging GTN resources. A thriving global community maintains these software and associated training resources to empower researcher-driven analyses. Collectively, the community-driven training activities offer a multitude of powerful onboarding mechanisms which lower the entry barrier for new users.

Read more about the ‘Galaxy of Informatics Resources for MS-based Proteomics’ in a recent review by Mehta et al in Expert Reviews of Proteomics.


### Metaproteomics to profile the gut microbiome of healthy adults subjected to microgravity

A joint effort by the German Aerospace Center (DLR), the European Space Agency (ESA), and the National Aeronautics and Space Administration Agency (NASA) aimed to simulate the effects of microgravity in the physiology of healthy humans resulted in the Artificial Gravity Bed Rest - European Space Agency (AGBRESA) study.

In this study, 12 subjects were studied for 90 days, 60 of which were spent laying down at a -6-degree tilt, a position designed to replicate the upward fluid shift experienced by astronauts. Additionally, we tested the effect of centrifugation-based countermeasure protocols to evaluate the reversal of potential damage.

Through metaproteomics analysis, we explored shifts in both the taxonomic and functional composition of the gut's microbial community. Our findings revealed that specific microbes linked to recovery or healthy status increased in subjects undergoing spaceflight countermeasure conditions, while the opposite trend was observed in subjects exposed to challenging spaceflight simulations. This approach enhances our understanding of gut functionality by using a non-invasive methodology correlating the microbial composition of human stool samples with physiological status.
