---
title: May 2024 Newsletter
publishDate: "2024-05-08"
---

## From the co-administrators

Dear members of the Metaproteomics Initiative,

We are pleased to share the latest updates in the field of metaproteomics through our first newsletter of 2024. Over the past months, our community has witnessed significant progress driven by conferences, workshops, cutting-edge method developments, and insightful publications—sent to us by you, our members!
 
We look forward to seeing many of you at our upcoming webinar on June 11. Registration is still open, and we encourage you to secure your spot!

Kind regards, 
Tim Van Den Bossche and Benoit Kunath

---

## Executive Board Digest

### Lorentz Center Workshop "Deciphering Microbiome Functions"
The Lorentz Center Workshop on [‘Deciphering Microbiome Functions’](https://www.lorentzcenter.nl/deciphering-microbiome-functions.html) was held in Leiden, Netherlands from March 25th to March 28th, 2024. The workshop brought together 45 researchers from the microbiome community for a week of sharing, learning, and collaboration. The workshop featured a theme for each day, starting with Introduction (Day One), Metagenomics (Day Two), Metaproteomics (Day Three) and Metabolomics and Collaboration and Future Directions (Day Four). Each day began with an introduction to the field by thought leaders, followed by tutorials by experts and brainstorming sessions in the afternoon.

The Introduction Day (Monday) was led by Magnus Palmblad and this included welcome by Lorentz Center, an introduction to the workshop’s goals and format and one-slide introductions by all attendees. In the afternoon, a long brainstorming session  on ‘Understanding and synergy between omics’ allowed participants to identify key  challenges and opportunities for integrating multi-omics datasets. This was followed by two breakout sessions focused on ‘Metadata study design’ and ‘Knowledge Graphs for Multi Omics Data Integration’.

The Metagenomics Day (Tuesday), hosted by Tim Van Den Bossche, provided attendees with an introduction to metagenomics methods and tutorials while fostering discussions to deepen understanding of  microbiome functionality. The day started with a talk by Georg Zeller from the Leiden University Medical Center (NL), offering insights into microbiome research from the perspectives of metagenomics and metatranscriptomics. Later, Robert Finn from EMBL-EBI (UK) delivered a talk on the MGnify platform for metagenomics analysis.

![Proceedings of the second day of the Lorentz workshop](/images/05_2024_newsletter/lorentz1.png)

The breakout sessions continued the brainstorming on ‘Understanding and synergy between omics’, along with sessions focused on ‘Design of synthetic communities for microbiome function analysis.’ and ‘International Consortium on Precision Research and Innovation in Microbiome’. In the evening, Greg Caparaso from the Northern Arizona University (USA) delivered a talk on the “Microbiome Data Science with QIIME 2”. This was followed by a lively Pizza dinner accompanied by scientific discussions well into the evening, and also a Karaoke Session titled “Acoustic Microbes”, featuring enthusiastic performances salvaged by Velma Tea Essi Aho and Feng Xian, amongst others.

![Breakout sessions during the Lorentz workshop](/images/05_2024_newsletter/lorentz2.png)

The Metaproteomics Day (Wednesday), hosted by Lucia Grenga, started with an enlightening presentation by Robert Hettich from Oak Ridge National Laboratory (USA) offering an Introduction to microbiome research from a metaproteomics perspective. This was followed by a detailed tutorial on metaproteomics analysis delivered by Alfredo Blakeley-Ruiz, North Carolina State University (USA). During the afternoon breakout sessions, attendees engaged in a diverse array of discussions: a) Supporting interoperability of software tools for microbiome multi-omics research; b) Environmental Microbiome - Global Study; c) Mechanistic-based pathways to support deciphering microbiome functions; d) Is deciphering species specific functions possible?; e) Integrating metagenomics with metaproteomics for improved protein annotation, including proteins of unknown function (PUFs) and f) Challenges in analyzing large meta-omics datasets. Later in the evening, the workshop participants enjoyed a delightful walk through the old city center and a nice dinner at the Het Prentenkabinet.

![Group picture of all participants of the Lorentz workshop](/images/05_2024_newsletter/lorentz3.png)

The final Metabolomics and Collaboration and Future Directions Day (Thursday), led by Pratik Jagtap, had the goals of introducing metabolomics and foster collaboration opportunities. The day began with an introduction to microbiome research from a metabolomics perspective delivered  by Neha Garg, Georgia Tech (USA). This was followed by a tutorial on metabolomics analysis presented by Marnix Medema from Wageningen University. In the afternoon, Daniel Figeys from the University of Ottawa (Canada) delivered an appreciated closing talk on Microbiome Research in Clinical Studies. The breakout sessions on this final day  included a) Long reads and Metatranscriptomics and b) Planning workshop manuscript.

The organizers gathered positive feedback from the attendees and are currently following up with breakout session leads to pursue the discussion plans, collaborations and paper writing. The organizers themselves have also started up new collaborations with several participants.
 
From the organizers’ perspective, the workshop was highly successful in no small part thanks to the dedicated, professional and enthusiastic support from the Lorentz Center staff planning, leading up to and during the workshop. Organizing a workshop on this scale was only possible due to generous sponsorship by Bruker, FEMS, University of Ottawa-School of Pharmaceutical Sciences, Qiagen, New England BioLabs, Thermo Fisher Scientific, and financial support from the Metaproteomics Initiative acquired from HUPO.

### US-HUPO Conference in Portland, Oregon (USA) 

US-HUPO’s 20th Annual Conference was held in Portland (OR) with the theme of “Bridging ‘Omics to Function”. During the event, Dr. Tim Griffin and Dr. Pratik Jagtap hosted an evening workshop on metaproteomics titled “Human Microbiome and Clinical Metaproteomics”. The workshop was attended by about 20 researchers and topics such as addressing challenges in clinical metaproteomic studies were discussed along with workflows and an introduction to the Metaproteomics Initiative. Attendees engaged in a lively discussion, expressing interest in forming a “Special Interest Group” for clinical metaproteomics. US-HUPO also featured the release of an introductory metaproteomics video titled [“The Legend of Metaproteomics”](https://www.youtube.com/watch?v=i0kZSqbAlrY) as part of the US-HUPO’s Virtual Media Outreach Committee’s initiative to generate educative videos in the field of basic and advanced proteomics.

Additionally, Drs. Griffin and Pratik Jagtap presented two posters on clinical metaproteomics: “A novel microbe-host biomarker panel for early detection of ovarian cancer (OvCa) in routinely collected PAP test clinical samples” and “Investigating key host, microbial and variant peptides for detection of oral cancer using advanced multi-omics methods”. Overall, [the conference was an exciting experience](https://www.genengnews.com/topics/omics/proteomics-in-portland-a-video-update-from-the-us-hupo-conference/) marked by some excellent talks and insightful discussions on the status of human proteomics research. 

![US-HUPO Conference in Portland](/images/05_2024_newsletter/hupo.png)

---

## Save the Date

Both Critical Assessment of MetaProteome Investigation (CAMPI) are moving ahead. Here's the latest update.

### Metaproteomics Webinar 2024
![US-HUPO Conference in Portland](/images/05_2024_newsletter/webinar.png)

Register for the webinar [here](https://metaproteomics.org/symposia/webinar-2024/).

### 6th Metaproteomics Symposium
The 6th Metaproteomics Symposium is taking place in Oslo, Norway from January 13th until January 15th, 2025. [Click here](https://metaproteomics.org/symposia/sixth/) to learn more.

---

## Metaproteomics articles you don't want to miss

### Scaling Microbiome Drug Response Assay through Automated Metaproteomics

Our gut microbiome is a complex ecosystem that can be influenced by different substances that enters our colon – dietary components, therapeutic drugs, prebiotics and probiotics, etc. The development and application of a high-throughput metaproteome response screening approach of individual gut microbiomes accelerates drug discovery and our understanding of microbiome-drug interactions.

In a recent study published in Microbiome Research Reports, [F]igeys and colleagues](https://www.northomics.ca/) presented an approach named [RapidAIM 2.0](https://www.oaepublish.com/articles/mrr.2023.57). Built upon versions developed by the team, RapidAIM 2.0 realizes high-throughput microbiome culturing and drug treatment, automates protein digestion and desalting, introduces automated TMT multiplexing for efficient metaproteome analysis, and streamlines statistical analysis for response detection. With these improvements, the updated workflow significantly reduces analysis time, enabling faster and more efficient experimentation. The updated workflow achieves remarkable efficiency, processing 320 samples in just six days. With TMT multiplexing, it quantifies an impressive 5,000 protein groups per sample.


### An incredible density of information from a fecal microbiome with the latest generation of tandem mass spectrometers!

[Dumas and his co-authors](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01766-4) delineated the remarkable capabilities of the latest generation of tandem mass spectrometers. By operating at a staggering 200 Hz, these instruments yield signals of such density that their interpretation becomes a laborious task. To address this challenge, they propose a two-steps approach: initially, to "proteotype" (i.e., identify) the organisms within the sample, followed by the development of a tailored database encompassing only the most abundant organisms present. This enables swift functional profiling at the species or even sub-species level. 


### Enzymatic machinery of wood-inhabiting fungi that degrade temperate tree species

Deadwood provides habitat for fungi and serves diverse ecological functions in forests. We already have profound knowledge of fungal assembly processes, physiological and enzymatic activities, and resulting physico-chemical changes during deadwood decay. However, in situ detection and identification methods, fungal origins, and a mechanistic understanding of the main lignocellulolytic enzymes are lacking.

[This study](https://academic.oup.com/ismej/article/18/1/wrae050/7633912) used metaproteomics to detect the main extracellular lignocellulolytic enzymes in 12 tree species in a temperate forest that have decomposed for 8 ½ years. Mainly white-rot (and few brown-rot) Basidiomycota were identified as the main wood decomposers, with Armillaria as the dominant genus; additionally, several soft-rot xylariaceous Ascomycota were identified. The key enzymes involved in lignocellulolysis included manganese peroxidase, peroxide-producing alcohol oxidases, laccase, diverse glycoside hydrolases (cellulase, glucosidase, xylanase), esterases, and lytic polysaccharide monooxygenases. The fungal community and enzyme composition differed among the 12 tree species. Ascomycota species were more prevalent in angiosperm logs than in gymnosperm logs. Regarding lignocellulolysis as a function, the extracellular enzyme toolbox acted simultaneously and was interrelated (e.g. peroxidases and peroxide-producing enzymes were strongly correlated), highly functionally redundant, and present in all logs.

In summary, our in situ study provides comprehensive and detailed insight into the enzymatic machinery of wood-inhabiting fungi in temperate tree species. These findings will allow us to relate changes in environmental factors to lignocellulolysis as an ecosystem function in the future.

### Metaproteomic assessment of gut microbial and host functional perturbations in Helicobacter pylori-infected patients subjected to an antimicrobial protocol

The impact of therapeutic interventions on the human gut microbiota (GM) is a clinical issue of paramount interest given the strong interconnection between microbial dynamics and human health. Orally administered antibiotics are known to reduce GM biomass and modify GM taxonomic profile. However, the impact of antimicrobial therapies on GM functions and biochemical pathways has scarcely been studied.

[Here](https://www.tandfonline.com/doi/full/10.1080/19490976.2023.2291170), we characterized the fecal metaproteome of 10 Helicobacter pylori-infected patients before (T0) and after 10 days (T1) of a successful quadruple therapy (bismuth, tetracycline, metronidazole, and rabeprazole) and 30 days after therapy cessation (T2), to investigate how GM and host functions change during the eradication and healing processes.

At T1, the abundance ratio between microbial and host proteins was reversed compared with that at T0 and T2. Several pathobionts (including Klebsiella, Proteus, Enterococcus, Muribaculum, and Enterocloster) were increased at T1. Therapy reshaped the relative contributions of the functions required to produce acetate, propionate, and butyrate. Proteins related to the uptake and processing of complex glycans were increased. 

Microbial cross-feeding with sialic acid, fucose, and rhamnose was enhanced, whereas hydrogen sulfide production was reduced. Finally, microbial proteins involved in antibiotic resistance and inflammation were more abundant after therapy. Moreover, a reduction in host proteins with known roles in inflammation and H. pylori-mediated carcinogenesis was observed. In conclusion, our results support the use of metaproteomics to monitor drug-induced remodeling of GM and host functions, opening the way for investigating new antimicrobial therapies aimed at preserving gut environmental homeostasis.

### A computational workflow for high throughput DIA-MS based metaproteomics analysis

Analyzing large-scale DIA-MS metaproteomic data remains a computationally challenging due to resource demands and elevated false positive identifications. We introduce metaExpertPro, a workflow for efficiently constructing spectral libraries and analyzing extensive DIA-based metaproteomics datasets.

[metaExpertPro](https://www.biorxiv.org/content/10.1101/2023.11.29.569331v1.full) integrates FragPipe and DIA-NN, facilitating spectral library generation from DDA-MS data and enabling peptide and protein quantification using DIA-MS data from Thermo or Bruker MS instruments. Additionally, it includes functional and taxonomic annotation, as well as quantitative matrix generation for microbiota and hosts. The workflow is available in a user-friendly [Docker container](https://github.com/guomics-lab/metaExpertPro).

In performance tests with human fecal samples, metaExpertPro quantified an average of 45,000 peptides in a 60-minute diaPASEF injection, surpassing three existing software tools in peptide and protein characterization. Notably, metaExpertPro maintained a low factual FDR of less than 5% for protein groups in four benchmark tests.

metaExpertPro facilitates large-cohort clinical research. We are actively working on a thousand-person metaproteomics cohort study on gut microbiota, aiming to utilize metaproteomics to elucidate the protein expression patterns and functional profiles of the human gut microbiota in a Chinese population, and uncover their significant roles in metabolic diseases

### MetaDIA: A Novel Database Reduction Strategy for DIA Human Gut Metaproteomics

Recent advancements in proteomics technology, especially in data-independent acquisition (DIA), have revolutionized our capacity for depth and accuracy in protein identification and quantification. This shift from data-dependent acquisition (DDA) to DIA is becoming increasingly prevalent. However, the full potential of DIA technology has not been realized in microbiome research. The primary challenge lies in the inherent requirements of metaproteomics for matching peptides to mass spectra within a vast and complex search space. DIA, due to the complexity of its data structure, requires a constrained search space. Researchers who want to apply DIA with microbiome samples were all compelled to use a spectral library derived from identification DDA. This method is not only resource-intensive but also limits the scope of peptide identification. 

[Here](https://www.biorxiv.org/content/10.1101/2024.03.14.585104v1), we’re excited to introduce a novel strategy designed to refine the search space for DIA analysis in metaproteomics. We score each individual peptide and prioritize those most likely to be detected. The final database only contain up to around 1 million peptides. Based on this strategy, we have developed and optimized a workflow called MetaDIA for analysis of microbiome DIA data, which operates independently of DDA assistance. We compared our method with the conventional DDA-based method and demonstrated high consistency between the two approaches.

We believe that our study will promote the application of DIA in metaproteomics and can facilitate the more accurate and comprehensive analysis of microbiome samples.
