---
title: March 2023 Newsletter
---

## From the co-administrators

Dear member of of the Metaproteomics Initiative,

We are thrilled to present to you the second edition of our newsletter, filled with exciting news and updates in the world of metaproteomics. The goal of this newsletter remains the same - to enhance communication and interaction among community members by sharing valuable information, latest research trends and upcoming events.

In this edition, we bring you a collection of summaries from our monthly meetings, upcoming metaproteomics sessions at international conferences as well as research highlights.

We are glad to receive more and more content from your side and to increase our interactions, which will be one of the key aspects of the coming metaproteomics symposium. We hope that many of you can join and engage with us regarding the Initiative and its projects.

Thank you for your continued support and participation in the Metaproteomics Initiative.

We look forward to reading more from you (and seeing you at IMS2023),
Tim Van Den Bossche and Benoit Kunath

---

## Executive Board Digest

We are exicited to share that both Critical Assessment of MetaProteome Investigation (CAMPI) are moving ahead. We believe that these projects will help move the field of metaproteomics forward, and we hope that many of you will be able to participate in them.

### CAMPI-2: Sample handling
[CAMPI-2](https://metaproteomics.org/campi/campi2/) has led to a collaboration between 10 laboratories in Europe and the US, where five protocols for metaproteome stabilization are being compared. These protocols have been designed for two different microbiota samples: human feces and soil. This will provide a standard procedure for sample handling, one of the main challenges in metaproteomics studies.

### CAMPI-3: Taxonomic and Functional Annotation of Metaproteomics Data
The CAMPI-3 team is busy pre-analyzing MS data from low-complexity and high-complexity samples. They are also developing reporting metrics for participants and will promote the study during the upcoming [5th International Metaproteomics Symposium](https://ims23.com/) and [ASMS Conference](https://www.asms.org/conferences/annual-conference/annual-conference-homepage), providing an opportunity for the metaproteomics research community to participate.

---

## Houston, We Have a (Microbiome) Problem…
(...And how the Metaproteomics Initiative will solve it!)

Pratik Jagtap, Robert Hettich, Tim van den Bossche and Timothy Griffin will be conducting an Evening Workshop at the [71st ASMS Conference](https://www.asms.org/conferences/annual-conference/annual-conference-homepage) on Mass Spectrometry and Allied Topics to be held in Houston from June 4 - 8, 2023. The workshop will be  at 5:45 PM local time on 7th June 2023.

Mass spectrometry-based metaproteomics research has experienced rapid growth due to its ability to help characterize complex microbial communities and is likely to become a central approach for understanding how microbiomes function. Despite its value, metaproteomics offers analytical and bioinformatic challenges beyond those encountered in traditional, single-organism MS-base proteomics.

As a solution, the Metaproteomics Initiative is a global initiative that promotes the dissemination of metaproteomics fundamentals, analytical and bioinformatic advancements, and microbiome applications. Members of the Initiative will present the updates on two recent CAMPI (Critical Assessment of Metaproteome Investigation) benchmark studies on sample preparation and functional annotations, and future projects that will be designed to propel this field forward. The members will also provide highlights from the 5th International Metaproteomics Symposium, which was held in April 2023 in Avignon, France.

The workshop will invite a panel of leading metaproteomics experts who will participate in a discussion that covers the status of metaproteomics, research needs, and growth opportunities. The panel will also interact with the audience and address discussion points on how the initiative will help in gaining deeper insights into microbiome dynamics. The workshop will also describe the Metaproteomics Initiative along with information on how interested researchers can join, participate, and contribute to its growth.

---

## CAMPI-3 Taxonomic and Functional Annotation of Metaproteomics Data

Metaproteomics plays a crucial role in characterizing microbiome functions. The acquired MS/MS data is searched against a protein sequence database to identify peptides, which are then used to infer a list of proteins in a metaproteome sample. Protein inference remains a major challenge for metaproteomics because of the many degenerate peptides shared among homologous proteins in different organisms.

The CAMPI-3 project will reach out to metaproteomics researchers with microbial datasets of low and high complexity and seek answers regarding the annotation of taxonomy, protein inference, and functional annotation.

CAMPI-3 will be carried out in four phases: A) Sample Preparation, B) Pre-Analysis, C) Promotions, and D) Analysis phase. Following their preparation, a microbial dataset from a synthetic mixture (low-complexity sample) and a high-complexity gut microbiome/wastewater dataset will be pre-analyzed to assess the data quality and define the metrics to report taxonomic and functional annotation results. Then the study will be promoted and both datasets provided, along with guidelines and points to be addressed, to the participating labs. In the last stage, the CAMPI-3 team will analyze the study participants' submissions and compare with the ground truth data (for the low-complexity data) and methods and results for the high-complexity dataset. Currently, CAMPI-3 is in the sample preparation phase. We anticipate it to be completed by the end of 2023, with the outcome of the study made available to the microbiome community.

---

## Save the Date

### The 5th International Metaproteomics Symposium

The deadline for registering in the [the 5th Metaproteomics Symposium](http://ims23.com/) is just days away!

The program is designed to facilitate collaboration and community-led brainstorming, with six sessions on various metaproteomics application fields/ aspects, two round tables, two additional sessions dedicated to collaborative projects and the Metaproteomics Initiative and numerous opportunities for attendees to present their work. The breaks, poster sessions, and a social event are designed to encourage networking and interactions among the attendees.

And that's not all!

Two satellite workshops are also organized to provide training on metaproteomics analysis using the Galaxy platform and other informatics resources available for functional microbiome research ([Workshop 1](https://ims23.com/workshop1/)) or an hands-on metaproteomics sample preparation and MS tips ([Workshop 2](https://ims23.com/workshop2/)).

Don't miss out on this unique opportunity to learn from the leading experts in the field, share your work, and network with your peers. We look forward to welcoming you in Avignon for the 5th International Metaproteomics Symposium.

### International Proteomics Summer School 2023

The Institute of Microbiology offers an annual [International Proteomics Summer School](https://mikrobiologie.uni-greifswald.de/en/teaching/proteomics-summer-school/international-proteomics-summer-school/). The one-week course (10-15 July 2023) organized by Prof. Dörte Becher & Dr. Susanne Sievers includes theoretical and lab sessions.

Main Topics
* Experimental design (theory, including individual coaching)
* Protein extraction &  sample preparation (theory & practical experience)
* Applications in microbiology cellular and meta-proteomics (theory)
* Mass spectrometry-based approaches (theory & practical experience)
* Label-based and label-free quantitative proteomics (theory & practical experience)
* Data analysis & visualization (theory & practical experience)

Contact & Registration

Due to limited lab space the number of participants is restricted. The application deadline is 15 May 2023.

Dr. Sandra Maaß<br>
phone: +49 3834 420 5921<br>
e-mail: sandra.maass@uni-greifswald.de

---

## Metaproteomics articles you don't want to miss

### Metaproteomics sheds bright light on dark matter within the rumen microbiome!

Thea Os Andersen (PhD student) et al, from the [Microbial Ecology and Meta-Omics (MEMO) group](https://www.nmbu.no/en/research/groups/memo) at The Norwegian University of Life Sciences (NMBU), have recently investigated the metabolic influence of rumen protozoa, which can constitute up to 50% of the microbial biomass in the rumen, but compared to bacteria and archaea, are sparsely studied. The predominant reason they are poorly described in vivo is due to difficulties of axenically culturing rumen protozoa as well as their complex genomic features that are obstinate to current metagenomic assembly and binning technologies. This recently changed in August 2022, with [single-cell amplified genomes (SAGs) being recovered from rumen microbiome samples](https://www.nature.com/articles/s41396-022-01306-8), representative of 5 holotrich and 14 entodiniomorph species spanning 13 genera. Another fun fact when it comes to rumen protozoa is that, based on initial screens the MEMO group have done so far, many species are structurally conserved across different individual animal hosts and even breeds, meaning we can sub in these new protozoal SAGs into our metaproteome search databases and detect their proteins! This is in stark contrast to bacteria and archaea for which there is a LOT of variation across individual animals meaning sample-specific databases are absolutely needed to capture their proteomes. All in all, the MEMO group has been eager to introduce these new protozoal genomes into their sample-specific metaproteomic pipelines so that we can simultaneously detect eukaryotic AND prokaryotic populations in the rumen microbiome.

For Thea, the inclusion of these SAGs nearly tripled her database size from 450k to ~1.2 million protein entries and required a different software (FragPipe instead of MaxQuant) to detect and quantify proteins. By including the SAGs, Thea also observed a huge boost in the identification rates, which in some animals was pushing 60%! Initial proteome estimations via total protein counts and label-free quantification highlighted that protozoal species indeed comprise an extensive fraction of the total rumen metaproteome of both cattle and goats that in some instances was greater than that of bacteria! Higher proteomic detection of protozoa occurred in animals that emitted more methane gas, particularly related to metabolisms that digest plant fibers and produce hydrogen (a fermentation intermediate used by methanogenic organisms). Thea also reaffirmed her observations in geographically independent datasets, underlining the substantial metabolic influence that under-explored protozoal populations have in the rumen, with greater implications for both digestion and methane metabolism. Check out the pre-print [here](https://www.biorxiv.org/content/10.1101/2022.06.22.497163v5).

As a final note, the explosion of molecular technologies that are rapidly improving genome resolution are having a huge influence on metaproteomic search databases, which for the rumen have expanded from 0.5 to over 4 million gene entries just in the past months! Exciting times ahead!

### Unipept Desktop 2.0: construction of targeted reference protein databases for proteogenomics analyses

[We](https://unipept.ugent.be/about) are thrilled to announce the release of Unipept Desktop 2.0, the latest version of the Unipept Desktop tool that is powering metaproteomic analyses.

Our updated tool now supports the automatic construction of targeted protein reference databases that only contain proteins associated with a predetermined list of taxa. If an approximate taxonomic composition of a sample is known (by, for example, performing a meta genomics experiment), you can greatly improve both the taxonomic and functional resolution of your analysis, giving you unprecedented insights into your data.

Since the protein reference databases that are being used for the analyses now reside on your local machine, you have full control over the database that’s being used. It also enables you to enjoy faster analyses times and a greater peace of mind since possibly confidential and privacy-sensitive data no longer needs to be transmitted over the internet.

As a proof of concept, we have performed a case study in which a human gut metaproteome dataset is analysed with Unipept Desktop 2.0 using different targeted databases on matched 16S rRNA gene sequencing data. Full details on this case study and how Unipept Desktop 2.0 works in detail can be found in [our manuscript](https://www.biorxiv.org/content/10.1101/2023.02.09.527820v1), available on BioRXiv.

The latest version of Unipept Desktop can be downloaded from our website: [https://unipept.ugent.be/desktop#download](https://unipept.ugent.be/desktop#download).
